Solutions/IronNet IronDefense/Playbooks/IronNet_UpdateIronDefenseAlerts/azuredeploy.json

```diff
@@ -98,73 +102,81 @@
 							"defaultValue": {},
 							"type": "Object"
 						},
-						"IronNetUrl": {
-							"defaultValue": "[parameters('IronDefenseUrl')]",
-							"type": "SecureString"
-						},
-						"Password": {
-							"defaultValue": "[parameters('IronApiPassword')]",
-							"type": "SecureString"
+						"tenant_id": {
+							"defaultValue": "[subscription().tenantId]",
+							"type": "String"
 						},
-						"TenantId": {
-							"defaultValue": "[parameters('TenantId')]",
-							"type": "SecureString"
+						"irondefense_url": {
+							"defaultValue": "[parameters('IronDefenseUrl')]",
+							"type": "String"
 						},
-						"Username": {
+						"user_name": {
 							"defaultValue": "[parameters('IronApiUsername')]",
+							"type": "String"
+						},
+						"password": {
+							"defaultValue": "[parameters('IronApiPassword')]",
 							"type": "SecureString"
 						},
 						"client_id": {
-							"defaultValue": "[parameters('ClientId')]",
-							"type": "SecureString"
+							"defaultValue": "[parameters('AzureClientId')]",
+							"type": "String"
 						},
 						"client_secret": {
-							"defaultValue": "[parameters('ClientSecret')]",
+							"defaultValue": "[parameters('AzureClientSecret')]",
 							"type": "SecureString"
 						},
-						"ResourceGroupId": {
+						"resource_group_id": {
 							"defaultValue": "[resourceGroup().id]",
-							"type": "SecureString"
+							"type": "String"
 						},
-						"ResourceGroup": {
+						"resource_group": {
 							"defaultValue": "[resourceGroup().name]",
 							"type": "String"
 						},
-						"SubscriptionId": {
+						"subscription_id": {
 							"defaultValue": "[subscription().subscriptionId]",
 							"type": "String"
 						},
 						"workspace_name": {
-							"defaultValue": "[parameters('WorkspaceName')]",
-							"type": "SecureString"
+							"defaultValue": "[parameters('workspace')]",
+							"type": "String"
 						},
-						"Pollingtime": {
-							"defaultValue": "[parameters('param_recurrence_interval')]",
-							"type": "Int"
+						"polling_interval": {
+							"defaultValue": "[parameters('RecurrenceInterval')]",
+							"type": "String"
 						},
-						"PollingtimeFrequency": {
-							"defaultValue": "[parameters('param_recurrence_frequency')]",
+						"polling_frequency": {
+							"defaultValue": "[parameters('RecurrenceFrequency')]",
 							"type": "SecureString"
 						},
-						"ShareCommentWithIronDome": {
-							"defaultValue": false,
+						"share_comment_with_irondome": {
+							"defaultValue": "[parameters('ShareCommentWithIrondome')]",
 							"type": "Bool"
+						},
+						"microsoft_login_uri": {
+							"type": "string",
+							"defaultValue": "[concat('login.microsoft','online.com')]"
```

Comment:
Hello @ling-tx, Can you please declare a paramter for this and use it here, In that parameter's default value set as [concat('login.microsoft','online.com')]

Same is the case with other concanited strings as well.  Thanks

Reply:
I declare parameters but still have concat error